benchmarks/ridge/datasets/simulated_blobs.py

```diff
@@ -0,0 +1,42 @@
+from benchopt import BaseDataset, safe_import_context
+from benchopt.datasets import make_correlated_data
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+
+
+class Dataset(BaseDataset):
+    name = "Simulated_correlated_data"
+
+    parameters = {
+        "n_samples, n_features": [
+            (20_000_000, 100),
+            (5000, 5000),
+            (20_000_000, 10),
+            (2_000_000, 100),
+            (1500, 1500),
+        ],
+        "n_targets": [1, 5, 20],
```

Comment:
I tuned down dataset size and remove (1500, 1500). (1000000, 100) seems yet a bit too fast I went for  (5000000, 100) for now. I'd also like to be able to compare the fastest backend and it might be harder to see the differences without enough data..